src/Sepes.Infrastructure/Service/CloudResourceOperation/CloudResourceOperationCreateService.cs

```diff
@@ -71,18 +55,68 @@ public async Task<CloudResourceOperationDto> CreateUpdateOperationAsync(int sand
                 Description = AzureResourceUtil.CreateDescriptionForResourceOperation(sandboxResourceFromDb.ResourceType, CloudResourceOperationType.UPDATE, sandboxResourceId),
                 BatchId = batchId,
                 OperationType = CloudResourceOperationType.UPDATE,
+                Status = CloudResourceOperationState.NEW,
                 CreatedBy = currentUser.UserName,
                 CreatedBySessionId = _requestIdService.GetRequestId(),
                 DependsOnOperationId = dependsOn != 0 ? dependsOn : default(int?),
                 MaxTryCount = CloudResourceConstants.RESOURCE_MAX_TRY_COUNT
             };
 
-
             sandboxResourceFromDb.Operations.Add(newOperation);
             await _db.SaveChangesAsync();
             return await GetOperationDtoInternal(newOperation.Id);
         }
 
+        async Task<CloudResourceOperation> CheckIfOperationsToWaitFor(CloudResource resource, UserDto currentUser)
```

Comment:
General questions: Did you leave out a word in this function name to make it shorter? Maybe "CheckIfAnyOperationsToWaitFor" would make more sense